src/server/views/footer.pug

```diff
@@ -0,0 +1,35 @@
+footer.text-center
+	p &copy; 2016 
+		a(href='http://www.isis.vanderbilt.edu/', target='_blank') Institute for Software Integrated Systems
+		|  / 
+		a(href='http://www.vanderbilt.edu', target='_blank') Vanderbilt University
+		|  / 
+		a(href='http://www.vuse.vanderbilt.edu', target='_blank') School of Engineering
+	#login-modal.modal.fade(tabindex='-1', role='dialog', aria-labelledby='myModalLabel', aria-hidden='true', style='display: none;')
+		.modal-dialog
+			.loginmodal-container
+				h1 Login to NetsBlox
+				br
+				form(method='post')
+					input(type='text', name='username', placeholder='Username')
+					input(type='password', name='password', placeholder='Password')
+					button.login.loginmodal-submit(name='login', type='submit') Login
+					//- .login-help
+					//-   a(href='https://editor.netsblox.org/') Register
+					//-   |  - 
+					//-   a(href='https://editor.netsblox.org/') Forgot Password
+	script(src='https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js')
+	script(src='https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js', integrity='sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa', crossorigin='anonymous')
+	link(href='https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css', rel='stylesheet', integrity='sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN', crossorigin='anonymous')
+	script(src='https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js')
+	script(src='https://cdnjs.cloudflare.com/ajax/libs/lightslider/1.1.6/js/lightslider.min.js')
+	script(src='scripts/helper.js')
+	script(src='scripts/js-cookie.js')
+	script(src='scripts/main.js')
+	script.
+		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+		ga('create', 'UA-62655804-1', 'auto');
```

Comment:
It would be nice if the google analytics key was not in version control and was provided by the server from an env variable (this is not the right one - the correct one is only on aws)